src/sentry/stacktraces/processing.py

```diff
@@ -325,10 +341,24 @@ def normalize_stacktraces_for_grouping(data, grouping_config=None) -> None:
                     frames, platform, exception_data
                 )
 
-    # normalize in-app
+    # normalize `in_app` values, noting whether event has a mix of in-app and system frames
     with sentry_sdk.start_span(op=op, description="normalize_in_app_stacktraces"):
+        frame_mixes = {"mixed": 0, "in-app-only": 0, "system-only": 0}
+
         for frames in stacktrace_frames:
-            _normalize_in_app(frames)
+            stacktrace_frame_mix = _normalize_in_app(frames)
+            frame_mixes[stacktrace_frame_mix] += 1
+
+    # store frame mix data on event for use in metrics
+    event_metadata = data.get("metadata") or {}
+    event_metadata["in_app_frame_mix"] = (
```

Comment:
Could you please add a note that this will be use in group creation to track a metric?

Reply:
Done here: https://github.com/getsentry/sentry/pull/54453/files#diff-a5fb5bd743c57f79634528b0f94eef6cbe79295ab61d1f5051f310b8b5323442R344-R345

https://github.com/getsentry/sentry/blob/99f7937bea5d6d7851f55c968804fe8406d60ab2/src/sentry/stacktraces/processing.py#L344-L345